### DISCUSSION TITLE
Universe 2024 Workshop: Let's build an app with GitHub Copilot in 40 minutes! - Wednesday, Oct 30 10:30 AM - 11:10 AM PDT

### DISCUSSION THREAD

**Discussion by bryantson** [community/community#142536] [Other Feature Feedback, Questions, & Ideas]
## Universe 2024 Workshop: Let's build an app with GitHub Copilot in 40 minutes!

![image](https://github.com/user-attachments/assets/da187cac-df93-49eb-86cb-8dae2ec63357)

Use **GitHub Copilot** as your AI pair programmer to develop a real app with little time and information. In this session, **Bryant Jimin Son** and **Ari LiVigni**, both Senior Cloud Solutions Architects from the GitHub Expert Services team, will walk through a real-world use case: building a fitness app from end to end. Together, you'll create a full stack of technologies for the infrastructure, frontend, and backend using the latest Copilot features.

## Speakers
- Ari LiVigni (@arilivigni), Sr. Cloud Solutions Architect, GitHub
- Bryant Jimin Son (@bryantson) , Senior Cloud Solutions Architect, GitHub

## Session Registration:
[Follow this link to register and add to your agenda!](https://reg.githubuniverse.com/flow/github/universe24/attendee-portal/page/sessioncatalog/session/1715206042531001U3sI)

## Take home resources

If you want to try out later, here are resources you can follow along. We showed you an environment with Codespace during the workshop, but here are two ways where you can try. This will have both Codespace and Non-Codespace (Coming Soon) versions.

[Checkout this tutorial- gh.io/octofit-tutorial-codespace](https://gh.io/octofit-tutorial-codespace)

## Date/Time
- Wednesday, Oct 30 - 10:30 AM - 11:10 AM PDT

## Location (if in-person)
Sandbox Stage @ General's Residence

**Comment by manherrera** [comment]
Hi

**Comment by subba2048** [comment]
Hey
